components/stock-operation-modal/index.js

```diff
@@ -9,45 +13,103 @@ const StockOperationModal = (props) => {
     showModal,
     transactionType,
     stockId,
+    availableQty,
   } = props;
 
   const [quantity, setQuantity] = useState('');
+  const [isUserLoggedIn, setIsUserLoggedIn] = useState('');
```

Comment:
by default can we consider that user is not logged in, so `isUserLoggedIn` can be `false` by default?

Reply:
Right!